core/genesis.go

```diff
@@ -240,12 +240,21 @@ func SetupGenesisBlock(db ethdb.Database, genesis *Genesis) (*params.ChainConfig
 
 	// Check config compatibility and write the config. Compatibility errors
 	// are returned to the caller unless we're already at block zero.
-	headBlock := rawdb.ReadHeadBlock(db)
-	if headBlock == nil {
-		return newcfg, fmt.Errorf("missing head block")
+	// headBlock := rawdb.ReadHeadBlock(db)
+	// if headBlock == nil {
+	// 	return newcfg, fmt.Errorf("missing head block")
+	// }
+
```

Comment:
Can we remove this commented out code?

Reply:
again I intentionally left it as comment out to show difference between actual behaviour vs modified one. but since this was asked twice, I guess this looks ugly :D deleting it.

Reply:
Thanks for removing